preconditions/README.md

```diff
@@ -0,0 +1,58 @@
+# IOTICS PRECONDITIONS
+
+In order to guarantee the full streaming functionality of IOTICS, we would need:
+- **HTTP2** to be enabled with your environment;
```

Comment:
TLS 1.3 (we'll move to it soon) as a separate bullet point.
also - do we need to document/present anything about our certificates or CA?

Reply:
Good point on the certs, that hadn't stuck with me at all.
The way Corp proxies work (I think) is that the browsers on the corp hardware have the proxy setup as a trusted CA so when it "reissues" the cert for the secure site, it looks good.

If we allow the API to use proxies, I'm assuming it will need to do the same.
looks like we can override & augment the CA_bundle on request.get calls, but I'm assuming we'd need to do this at a system level (or delegate that the client figure that out)